<launch>
  <arg name="data_folder" default="/home/volcanoh/macaca/data/handeye/eyetracker"/>
  <arg name="rate" default="10"/>
  <node pkg="macaca_tf_setup" type="macaca_tf_setup_node" name="macaca_tf_setup" output="screen">
    <param name="scene_left_tf_datafile" type="str" value="$(arg data_folder)/scene/left/CalibratedTransform.yml"/>
    <param name="scene_right_tf_datafile" type="str" value="$(arg data_folder)/scene/right/CalibratedTransform.yml"/>
    <param name="eye_left_tf_datafile" type="str" value="$(arg data_folder)/eye/left/CalibratedTransform.yml"/>
    <param name="eye_right_tf_datafile" type="str" value="$(arg data_folder)/eye/right/CalibratedTransform.yml"/>
    <param name="rate" type="int" value="$(arg rate)"/>
  </node>
</launch>
